microsoft-edge/devtools-guide-chromium/progressive-web-apps/simulate-window-controls-overlay.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Display content in a PWA's title bar using Window Controls Overlay
+description: Display content in a PWA's title bar using Window Controls Overlay.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 05/17/2024
+---
+# Display content in a PWA's title bar using Window Controls Overlay
```

Comment:
best title of article?  describe the result as a static, simulated image of window controsl, per latest draft of article

Reply:
Latest titles: 
Simulate a Window Controls Overlay in a PWA's title bar area
Display content in the title bar area using Window Controls Overlay